docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -47,11 +29,17 @@ $ tctl tokens add \
     --app-uri=http://localhost:3000
 ```
 
-### TLS requirements
+### TLS Management
```

Comment:
If we have already included the `edition-prereqs-tabs.mdx` partial, I think we can delete this section, since the expectation is that the user already has a running Auth/Proxy service. 

Instead, we could add an `Admonition` for self-hosted Teleport users to make sure they have configured Teleport to use a certificate issue for a wildcard subdomain or application-specific sudomains. This could go in the "Application name" section.

Reply:
@ptgott this has been modified. Please review